src/server/resolvers.ts

```diff
@@ -205,6 +214,17 @@ export const resolvers: Resolvers = {
 		userType: () => UserType.Mentor,
 	},
 	Mutation: {
+		hackerStatus: async (root, { input: { id, status } }, { user, models }: Context) => {
```

Comment:
I think we need to make corresponding changes in the schema and regenerate the resolver for this?